.circleci/config.yml

```diff
@@ -2,9 +2,26 @@ version: 2.1
 orbs:
   architect: giantswarm/architect@5.11.4
 
+jobs:
```

Comment:
Already did a review for this file there https://github.com/giantswarm/observability-platform-api/pull/4